DecipherNow/gm-fabric-documentation
# fix: update deps and modify plugins to allow edits to lesson template

Resolves #69 

# How to Test
* Run app via `gatsby develop` and be sure to select bypass for network:
![image](https://user-images.githubusercontent.com/8700322/35226814-6d3c6878-ff5a-11e7-8abb-d8dac4d113fc.png)
* Modify a styled-component in /src/templates/Lesson.jsx` and ensure that the change hot loads without crashing WebPack.
* Run `npm run build:pp`
* Move /public/* to /public/gm-fabric-documentation/* and run http-server from /public/ and confirm that the static bundle loads as expected.
* Click around app and make sure things work as expected without the debugger getting triggered or without console.log errors.
